packages/blade/src/tokens/global/typography.ts

```diff
@@ -191,33 +191,33 @@ const typography: TypographyWithPlatforms = {
       size: {
         10: 10,
         25: 11,
-        50: 12,
-        75: 14,
+        50: 11,
+        75: 12,
         100: 15,
         200: 17,
         300: 18,
         400: 20,
         500: 22,
-        600: 24,
-        700: 27,
+        600: 22,
+        700: 24,
         800: 29,
         900: 32,
-        1000: 35,
+        1000: 29,
       },
       weight: {
         ...fontWeight,
       },
     },
     lineHeights: {
       s: 16,
-      m: 16,
+      m: 18,
       l: 24,
-      xl: 24,
-      '2xl': 28,
+      xl: 28,
+      '2xl': 24,
       '3xl': 28,
-      '4xl': 30,
+      '4xl': 32,
       '5xl': 30,
-      '6xl': 38,
+      '6xl': 40,
```

Comment:
yes that's something tragic since we have these tokens as aliases(common across native and web) and hence they are not very intuitive and not incrementing. not able to think of any other way 😢.

Reply:
Can we add some comment in the code to explain this? Because looking at tokens it seems very confusing that `xl` is `28` and `2xl` is smaller than it (`24`). Similarly for `4xl` and `5xl`